models/m1.smv

```diff
@@ -0,0 +1,6 @@
+MODULE main
```

Comment:
If these are reusable for all NuRV examples, please place them in `common/models/NuRV`. Otherwise, they go into `models/<example-name>/model`.